app/models/mixins/miq_provision_quota_mixin.rb

```diff
@@ -247,42 +253,88 @@ def quota_find_prov_request_by_group(options)
 
   def quota_find_active_prov_request_by_owner(options)
     email = get_option(:owner_email).to_s.strip
-    quota_find_active_prov_request(options).select { |p| email.casecmp(p.get_option(:owner_email).to_s.strip) == 0 }
+    quota_find_active_prov_request(options).select { |p| email.casecmp(p.request_owner_email(p)).zero? }
+  end
+
+  def request_owner_email(request)
+    service_request?(request) ? request.requester.email : request.get_option(:owner_email).to_s.strip
+  end
+
+  def service_request?(request)
+    request.type == "ServiceTemplateProvisionRequest"
   end
 
   def quota_find_active_prov_request_by_group(options)
-    prov_requests = []
-    prov_owner = get_owner
-    unless prov_owner.nil?
-      group = prov_owner.ldap_group
-      prov_requests = quota_find_active_prov_request(options).select do |p|
-        prov_req_owner = p.get_owner
-        prov_req_owner && group.casecmp(prov_req_owner.ldap_group) == 0
-      end
+    prov_request_group = miq_request.requester.current_group
```

Comment:
Knowing that `current_group` change change in the db for the user after the request is created should we base this on `miq_request.options[:requester_group]`?  (Note: This is the group name, not id)